content/terms/security.md

```diff
@@ -11,6 +11,9 @@ If you have specific questions or concerns, contact us at security@sourcegraph.c
 
 If you think you have discovered a security vulnerability in our product, please follow our instructions on [how to report a security vulnerability](https://handbook.sourcegraph.com/product-engineering/engineering/cloud/security/reporting-vulnerabilities).
 
+For details on our information security practices or to request a copy of our SOC 2 Report, please visit our 
+<Badge text="Security Trust Portal" color="blue" size="small" onClick={() => window.open('https://security.sourcegraph.com/')} />.
```

Comment:
Nice! I was thinking we'd have to use `Link` to wrap the Badge, but forgot about the `onClick` prop I added on Badge. This is a lot simpler! 😁 

Only thing I'd suggest is, if we're allowing this to open in a new tab, we should probably add `noopener` for security.

```suggestion
<Badge text="Security Trust Portal" color="blue" size="small" onClick={() => window.open('https://security.sourcegraph.com/', '_blank', 'noopener')} />.
```

Reply:
Good point! Yeah, the most light weight we can keep components will make our stakeholders lives easier for blog writing 😄 - thanks!